ImageDownloader.py

```diff
@@ -2,11 +2,16 @@
 
 import urllib
 import os
+from urlparse import urlparse
+from os.path import splitext,basename
+from multiprocessing import Pool,TimeoutError
 
 class ImagesDownload(object):
 
-	def __init__(self, fileName = None):
+	def __init__(self, pool_size=5,timeout=10,fileName = None):

```

Comment:
fileName should be first parameter to method because class is instantiated with 'ImageCollection.txt' file name